test/e2e/lib/framework/serverApps/mock.ts

```diff
@@ -116,34 +116,27 @@ export function createMockServerApp(servers: Servers, setup: string): MockServer
     res.end()
   })
 
-  app.get('/datadog-logs.js', (_req, res) => {
-    res.sendFile(sdkBuilds.LOGS_BUNDLE)
-  })
-
-  app.get('/datadog-rum.js', (_req, res) => {
-    res.sendFile(sdkBuilds.RUM_BUNDLE)
-  })
-
-  app.get('/chunks/:name-:hash-datadog-rum.js', (req, res) => {
-    const { name, hash } = req.params
-    res.sendFile(sdkBuilds.rumBundleRecorderChunk(name, hash))
-  })
-
-  app.get('/datadog-rum-slim.js', (_req, res) => {
-    res.sendFile(sdkBuilds.RUM_SLIM_BUNDLE)
-  })
+  app.get(/datadog-(?<packageName>[a-z-]*)\.js/, (req, res) => {
+    const { originalUrl, params } = req
 
-  app.get('/worker.js', (_req, res) => {
-    res.sendFile(sdkBuilds.WORKER_BUNDLE)
+    if (process.env.CI) {
+      res.sendFile(getSdkBundlePath(params.packageName, originalUrl))
+    } else {
+      forwardToDevServer(req.originalUrl, res)
+    }
   })
 
-  app.get('/app.js', (_req, res) => {
-    res.sendFile(sdkBuilds.NPM_BUNDLE)
+  app.get('/worker.js', (req, res) => {
+    if (process.env.CI) {
+      res.sendFile(getSdkBundlePath('worker', req.originalUrl))
+    } else {
+      forwardToDevServer(req.originalUrl, res)
+    }
   })
 
-  app.get('/chunks/:name-:hash-app.js', (req, res) => {
-    const { name, hash } = req.params
-    res.sendFile(sdkBuilds.npmBundleChunks(name, hash))
+  app.get(/(?<appName>app).js$/, (req, res) => {
```

Comment:
@RomanGaignault, I think you'll just need to add `react-app` to the regex:
```suggestion
  app.get(/(?<appName>app|react-app).js$/, (req, res) => {
```